test/lib/mayaUsd/render/pxrUsdMayaGL/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(TEST_SCRIPT_FILES
     testProxyShapeDuplicatePerformance.py
     testProxyShapeLiveSurface.py
     testProxyShapeRendererSceneMessages.py
+    testProxyShapeSelectionPerformance.py
```

Comment:
@mattyjams 

As a FYI, I noticed that both testProxyShapeLiveSurface and testProxyShapeSelectionPerformance fail on my Windows after syncing to the latest dev changes.

@kxl-adsk  I believe you mentioned the same failure on your side.  I don't think I notice these failures when I originally reviewed this PR but they show up now. 

Requires more investigation.

```
98% tests passed, 2 tests failed out of 127

Label Time Summary:
interactive          = 847.06 sec*proc (14 tests)
pxrUsdMayaGL         = 847.06 sec*proc (14 tests)
translators          = 643.22 sec*proc (52 tests)
ufe                  = 264.30 sec*proc (21 tests)
usdPreviewSurface    =  20.88 sec*proc (2 tests)

Total Test time (real) = 212.37 sec

The following tests FAILED:
         15 - testProxyShapeLiveSurface (Failed)
         17 - testProxyShapeSelectionPerformance (Failed)
```

Reply:
Ahh, sorry about that @HamedSabri-adsk. They've been passing for me on Linux. I briefly had access to a Windows machine before we left the studio, but I've since lost that. :/

Do you have any logs handy from failed runs of either of those tests? And I assume all of the other `pxrUsdMayaGL` tests are passing?

Just a wild guess without any more detail for the moment, but the one thing common to these two tests and absent from all the others is the shenanigans they pull dealing with PySide2 and shiboken2 so that they can emulate clicking in the viewports. Maybe there's something there that's problematic on Windows?

Let me know what I can do to help get to the bottom of that!

Reply:
Indeed I saw these when validating your change @mattyjams and before merging. I was confused because @HamedSabri-adsk run the tests and they were passing. The explanation I found is that we can be using builds of Maya with asserts enabled. When we do, these tests will cause Maya to assert when the application is exiting. This then causes the test to fail. You won't get it with the official build of Maya.

The assert seems to be complaining that there are still some shared GL widgets alive when we are releasing resources.

Reply:
Thanks for the extra info @kxl-adsk. Makes sense then why I might not be seeing those failed asserts.

I took a wild stab at addressing this in #777, so we can continue the investigation there.